static/js/components/EmploymentTab.js

```diff
@@ -1,21 +1,131 @@
 import React from 'react';
+import Button from 'react-mdl/lib/Button';
+import Grid from 'react-mdl/lib/Grid';
+import { Cell } from 'react-mdl/lib/Grid';
+
+import ProfileTab from "../util/ProfileTab";
+import { saveAndContinue } from '../util/profile_edit';
+
+class EmploymentTab extends ProfileTab {
+  constructor(props) {
+    super(props);
+    this.saveAndContinue = saveAndContinue.bind(this, '/dashboard', this.validation);
+    this.blankWorkHistoryEntry = {
+      id: null,
+      position: "",
+      industry: "",
+      company_name: "",
+      start_date: "",
+      end_date: "",
+      city: "",
+      country: "",
```

Comment:
so we don't have `state_or_territory` on the mockup ([here](https://github.com/mitodl/micromasters/issues/150), unless there's a more recent version I haven't seen) is this something we're adding?